build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
     id 'idea'
     id 'com.diffplug.spotless' version '5.5.1'
     id 'com.github.spotbugs' version '4.0.0'
-    id 'com.github.johnrengelman.shadow' version '7.0.0'
+    id 'com.github.johnrengelman.shadow' version '6.1.0'
```

Comment:
The 7.0.0 version of shadow seemed to be causing some issues with Cloud Build, but I couldn't figure out why. The Cloud Build run would fail because it claimed that the gradle version was not recent enough, despite it being more recent than what was required.

Downgrading this to the previous release fixed the issues, but I'm not sure if that's the right move.

Reply:
@ichengchang can you test this branch and make sure deploys via gh actions still work for this version of the shadow plugin?

Reply:
No issue with GHA deployment using shadow Plugin 6.1.0.